Dockerfile.template

```diff
@@ -252,6 +252,12 @@ RUN set -eux; \
 			export LIBS='-lucontext'; \
 			;; \
 	esac; \
+{{ ) else "" end -}}
+{{ if .version | . == "3.3.0" then ( -}}
+	# workaround crash on aarch64 Linux (e.g. Docker Desktop on macOS)
+	# https://bugs.ruby-lang.org/issues/20085#note-5
+	# https://github.com/ruby/ruby/pull/9371
+	export ASFLAGS="${ASFLAGS:-} -mbranch-protection=pac-ret"; \
```

Comment:
```suggestion
{{ if .version == "3.3.0" then ( -}}
	# workaround crash on arm64
	# https://bugs.ruby-lang.org/issues/20085
	# https://github.com/ruby/ruby/pull/9385 <- https://github.com/ruby/ruby/pull/9371
	wget -O 'arm64-fix.patch' 'https://github.com/ruby/ruby/commit/7f97e3540ce448b501bcbee15afac5f94bb22dd9.patch?full_index=1'; \
	echo '86bc65415fd62cb2272a4df249f39fb79db15617ad05c540e05a22f02eae73b3 *arm64-fix.patch' | sha256sum --check --strict; \
	patch -p1 -i arm64-fix.patch; \
	rm arm64-fix.patch; \
```

Reply:
Sounds good. Applied and updated corresponding Dockerfiles in cfdac1e2a0de73395dea8f048eae2541cb26715f !